src/NewTools-RewriterTools-Backend/StRewriterDemoRules.class.st

```diff
@@ -21,24 +21,14 @@ StRewriterDemoRules class >> defaultRule [
 
 { #category : 'api' }
 StRewriterDemoRules class >> demoRules [
-
-	^ {
-		  self simplifyToIfNilIfNotNilRule.
-		  self replaceAssertToAssertEmptyRule.
-		  self simpliflyConditionalToIsEmptyRule.
-		  self removeEqualsInAssertRule.
-		  self replaceDenyDenyEmptyRule.
-		  self replaceCharacterTabAsStringToStringTabRule.
-		  self simpliflyToIfEmptyIfNotEmptyRule.
-		  self removeIfNilRule.
-		  self replaceSelectToRejectRule.
-		  self removeAssignmentWithoutEffectRule.
-		  self replaceCollectFlattenToFlatCollectRule }
+ 
+	^ (Pragma allNamed: #rewriterRule) collect: [ :prg | prg method valueWithReceiver: nil]
```

Comment:
Would not be better to call the pragma #rewriterDemoRule?

Reply:
I was thinking it would be better to remove the Demo bit altogether as they look like legitimate rules to me - and the UI that invokes them - the ReWriteEditor calls them an example rule. So possibly example rule? Although would template make more sense? Then per my previous comment - rename the class to StRewriterTemplateRules. These are templates you can use to make code changes. I wanted to make this change so that libraries like CodeParadise could add to the list to help users update their code